README_PTBR.md

```diff
@@ -0,0 +1,114 @@
+[![Logisim-evolution](artwork/logisim-evolution-logo.svg)](https://github.com/logisim-evolution/logisim-evolution)
```

Comment:
I am not sure if there's point of translating the README and not translating the rest of md files. I think it'd be better to remove this one and just focus on app translation.

Reply:
You're right, there's no point having only one for now, should just focus in the app for now